tbx/project_styleguide/templates/patterns/pages/work/work_listing.html

```diff
@@ -1,16 +0,0 @@
-{% extends "patterns/base_page.html" %}
-{% load wagtailcore_tags wagtailimages_tags static %}
-
-{% block content %}
-
-    {% include "patterns/molecules/title-filters/title-filters.html" with item=page tags=related_services %}
-
-    {% include "patterns/organisms/work-listing/work-listing.html" with works=works %}
```

Comment:
Can this `patterns/organisms/work-listing/work-listing.html` file be deleted now?

Reply:
Just realised this is used here:

https://github.com/torchbox/torchbox.com/blob/2f0bca405a7c0ce0839c2da644d2883ea432e746/tbx/project_styleguide/templates/patterns/pages/team/team_detail.html#L21

So we can't delete it at this point